todo/src/containers/TodoList.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {toggleTodo} from '../actions';
+
+import './TodoList.scss';
+
+class TodoList extends Component {
+    
+    toggleTodo = e => {
+        e.target.classList.toggle('completed');
```

Comment:
this is one way to handle adding or removing the class name onto the actual todo item -- but even though it's synced up and functioning properly, there's an opening for the classname set this to not represent the actual `completed` status of the actual todo item. instead, you could add a ternary on the individual todo item being returned by the map method below that dynamically adds this className or not depending on the `todo.completed` boolean